Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -79,10 +88,9 @@ public void initialize() {
 
         // preset the IMU angles so it doesn't start on null since it will only later be read when turning
         IMUOriginalAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
-    }
 
-    // flag to say whether we should disable the correction system
-    boolean turnFlag = false;
+        servoGrabber.setPosition(Constants.GRABBER_INITIALIZE_POSITION);
```

Comment:
```suggestion
        servoGrabber.setPosition(Constants.GRABBER_INITIAL_POSITION);
```

Reply:
The recent change to Constants.GRABBER_INITIAL_POSITION broke the code, and won't compile.  You need to go back and make the same name change to the constants class.

You should always check to see if it compiles before committing the change.

Reply:
I changed again and re-comitted and pushed, along with a few other changes.